package/kedro_viz/models/utils.py

```diff
@@ -0,0 +1,38 @@
+"""`kedro_viz.models.utils` contains utility functions used in the `kedro_viz.models` package"""
+
+import logging
+
+from kedro.io import AbstractDataSet
+
+logger = logging.getLogger(__name__)
+
+
+def get_dataset_type(dataset: AbstractDataSet) -> str:
+    """Get the type of a dataset as a string-- the abbreviated name of the module to
```

Comment:
```suggestion
    """Get the type of a dataset as a string: the abbreviated name of the module to
```
(very minor I know... it just looked weird to me)

Reply:
You're right, fixed in [89f04d7](https://github.com/kedro-org/kedro-viz/pull/1232/commits/89f04d7c912cff11e00dbdf095a5d54ae8d67e05).